benchmarks/instruction_counts/execution/runner.py

```diff
@@ -0,0 +1,240 @@
+import math
```

Comment:
Please give the 1 line description of the file at the top for skimmers and lazy people like me :)